src/plugins/intel_gpu/tests/unit/test_cases/paged_attention_gpu_test.cpp

```diff
@@ -68,15 +68,16 @@ struct CacheRotationDescriptor {
 struct PagedAttentionManager {
     int num_heads;
     int head_size;
+    int v_head_size;
```

Comment:
nit:
Please replace all occurrences of `head_size` -> `k_head_size` in this test
```suggestion
    int k_head_size;
    int v_head_size;
```